auditlog/models/rule.py

```diff
@@ -364,11 +472,20 @@ def create_logs(self, uid, res_model, res_ids, method,
         http_session_model = self.env['auditlog.http.session']
         for res_id in res_ids:
             model_model = self.env[res_model]
+            # Do extra check for active_model situations where res_model
+            # is not preloaded in auditlog model_cache
+            model_id = self.pool._auditlog_model_cache.get(res_model, False)
```

Comment:
True, while I do agree with the rewrite, I would still like to make use of the caching dict to prevent unnecessary lookups afterwards.

I would propose something like:

`            if not self.pool._auditlog_model_cache.get(res_model):
                self.pool._auditlog_model_cache[res_model] = \
                    self.env['ir.model'].search([
                        ('model', '=', res_model)]).id`